build/windows/installer/livenessprobe/livenessprobe.cpp

```diff
@@ -16,6 +16,7 @@
 #define CERTIFICATE_RENEWAL_REQUIRED 0x00000003
 #define FLUENTDWINAKS_SERVICE_NOT_RUNNING 0x00000004
 #define NO_WINDOWS_AMA_MONAGENTCORE_PROCESS 0x00000005
+#define NO_TELEGRAF_PROCESS 0x00000006
 #define UNEXPECTED_ERROR 0xFFFFFFFF
```

Comment:
1. lets remove windows livenessprobe.cpp for now and until we get the telemetry to figure out how many clusters are impacted.
2. Windows
   -  Add telemetry to figure out how many clusters are impacted
   - write simple Go lang function to check telegraf is running or not
   - update telemetry.go,  
       if (env(TELEMETRY_CUSTOM_PROM_MONITOR_PODS) == "true" ) {
                isTelegrafRunning := isTelegrafRunng()
				telemetryDimensions["TelegrafNotRunning"] = "true"				
       }